cg/meta/workflow/mutant/mutant.py

```diff
@@ -24,6 +27,9 @@ def __init__(
     ):
         super().__init__(workflow=workflow, config=config)
         self.root_dir = config.mutant.root
+        self.quality_checker = MutantQualityController(
+            status_db=config.status_db_, lims=config.lims_api_
```

Comment:
Use the property, not the attribute, as the attribute may (and probably) be None
```suggestion
            status_db=config.status_db, lims=config.lims_api
```

Reply:
Changed!